.github/CONTRIBUTING.md

```diff
@@ -14,18 +15,18 @@ Pull requests to Rhino are welcome!
 
 
 ## Development Process
-1. We follow [Scrum](https://scrumguides.org/).
-2. All changes are introduced in pull requests, which must be peer-reviewed. The reviewer inspects
-   the code, tests the changes and checks them against the [DoD](#definition-of-done) before
-   approving.
-3. The `develop` branch is the base for our regular work. It is set as the "default" branch on
-   GitHub so that PRs automatically target it and `closes` keyword works in issue descriptions.
-4. The `main` branch is used for releases. We regularly merge `develop` into `main`, increment the
-   version number and tag a new release on GitHub.
-5. We follow the [Semantic Versioning](https://semver.org/) scheme.
 
+1. We follow [Scrum](https://scrumguides.org/).
```

Comment:
is this true now? I'm not sure :) This is for sure true in periods of intensified development

Reply:
I guess "it depends" and doesn't bring much useful information here, gonna remove this point :wink: